File: esportskicentar/src/main/java/tech/esc/esportskicentar/model/Klijent.java
package tech.esc.esportskicentar.model;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Data
@EqualsAndHashCode(callSuper = false)
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Klijent {
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Id
    @Column(name = "id_klijent")
    private int idKlijent;
    @Basic
    @Column(name = "ime")
    private String ime;
    @Basic
    @Column(name = "prezime")
    private String prezime;
    @Basic
    @Column(name = "broj_telefona")
    private String brojTelefona;
    @Basic
    @Column(name = "korisnicko_ime")
    private String korisnickoIme;
    @Basic
    @Column(name = "lozinka")
    private String lozinka;
    @Basic
    @Column(name = "email")
    private String email;
    @Basic
    @Column(name = "blokiran")
    private boolean blokiran;

}

File: esportskicentarapp/src/app/models/klijent.ts
export class Klijent {

    idKlijent!: number;
    ime: string | null;
    prezime: string | null;
    korisnickoIme: string | null;
    lozinka: string | null;
    brojTelefona: string | null;
    email: string | null;
    role : string | null;
    blokiran : boolean;

    constructor(name:string | null, lastname:string | null, username:string | null, password:string | null, 
                phoneNumber:string | null, email : string | null, role : string | null) {
        this.ime = name;
        this.prezime = lastname;
        this.korisnickoIme = username;
        this.lozinka = password;
        this.brojTelefona = phoneNumber;
        this.email = email;
        this.role = role;
        this.blokiran = false;
    }


}
